src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.props

```diff
@@ -2,6 +2,6 @@
 <Project>
```

Comment:
Let's not rename this file yet as the package hasn't been renamed, so this is breaking the NuGet convention and therefore will not be imported.